README.md

```diff
@@ -7,55 +7,65 @@ query (or query template) as a table.
+<img width="1036" alt="Screenshot 2021-03-09 at 18 59 07" src="https://user-images.githubusercontent.com/11739632/110516230-9508c200-8109-11eb-9fae-1218010597f6.png">
 
 ## URL parameters
 
-These are the URL parameters that this plugin accepts:
-
-- `queryId` (**required**, integer): query to run (e.g. `queryId=26`).
-- `sourceKey` (**required**, string): Key of the data-source to run the
-  query on (e.g. `sourceKey=b16e9ed5`).
-- `limit` (**optional**, integer): maximum number of results to display
-  (e.g. `limit=1000`).
-- `param_number_PARAM NAME` (**optional**, number): *for query
-  templates*, any numerical parameter of the template (e.g.
-  `param_number_my%20number=123`).
-- `param_string_PARAM NAME` (**optional**, string): *for query
-  templates*, any string parameter of the template (e.g.
-  `param_string_my%20param=abc`).
-- `param_ids_PARAM NAME` (**optional**, comma-separated list): *for
-  query templates*, any edgeset/nodeset parameter of the template (e.g.
-  `param_ids_my%20id%20list=1,2,3`).
-
-Full URL example (line breaks for readability):
-```
-https://linkurious.example.com/plugins/data-table
-  ?queryId=26
-  &sourceKey=b16e9ed5
-  &limit=1000
-  &param_string_city=Paris
+This plugin supports the following URL parameters in the query string:
+
+| Param | Type | Description | Example |
+| :-- | :-- | :-- | :-- |
+| `queryId`                     | integer (**required**)  | ID of the query to run. | `queryId=87` |
+| `sourceKey`                   | string (**required**)   | Key of the data-source to run the query on. | `sourceKey=b16e9ed5` |
+| `limit`                       | integer (**optional**)  | Maximum number of results to display. | `limit=500` |
+| `param_number_{{Encoded field name}}` | number (**optional**)   | *For query templates*, any numerical parameter of the template | `param_number_age=30` |
+| `param_string_{{Encoded field name}}` | string (**optional**)   | *For query templates*, any string parameter of the template | `param_string_city=Paris` |
+| `param_ids_{{Encoded field name}}`    | comma-separated list (**optional**)  | *For query templates*, any edgeset/nodeset parameter of the template | `param_ids_target_ids=1,50,12` |
+
+## Usage with standard queries
+
+In order to display the result of a standard query in a table:
+
+1. Create a standard READ query. For example: `MATCH (n) return n LIMIT 1000`.
+2. Note down the newly-created query ID, it will by the value of `queryId`. For example: `queryId=87`.
+3. Compose a valid data-table plugin URL and open it in a new tab.
+
+You can either manually complete and open the following URL:
+```
+<your LKE base URL>/plugins/table?queryId=87&sourceKey=<a data-source key>
+```
+Or you can create the following [Custom Action](https://doc.linkurio.us/user-manual/latest/custom-actions/):
+```
+{{baseUrl}}plugins/table?queryId=87&sourceKey={{sourceKey}}
```

Comment:
you can literally use this syntax instead of the one above to explain which URL to go to, and then also saying they can save it as CA